package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "bcrs-business-create-ui",
-  "version": "0.1.0",
+  "version": "0.2.0",
```

Comment:
Our little app is growing up so fast.. 😢

Reply:
Another sprint, another release!